modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,175 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with user-workload monitoring.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+
+* You have created the `cluster-monitoring-config ConfigMap` object.
```

Comment:
```suggestion
* You have cluster administrator permissions.
* You have created the `cluster-monitoring-config` config map.
```
A suggestion re permissions wording so that the word cluster isn't used 3 times 🙂 
+ formatting fix for the config map